constants/app.test.js

```diff
@@ -31,7 +31,8 @@ jest.mock("../src/windowHelpers", () => ({
   getCurrentOrigin: () => "https://localhost:3000",
 }));
 
-const TEST_CLIENT_ID = "https://pod.inrupt.com/appsteamdev/clientid.json";
+const TEST_CLIENT_ID =
+  "https://pod.inrupt.com/appsteamdev/clientid.example.json";
```

Comment:
This shouldn't be a real URL, and instead use:
```suggestion
  "https://inrupt.test/clientid.json";
```

The `.test` and `.example` TLDs are reserved for this purpose, and will never actually resolve to anything in particular: https://en.m.wikipedia.org/wiki/.test

This is what I meant by saying all URLs in tests need to be updated. /cc @chelseapinka